### ISSUE TITLE
how to setup html-settings

### ISSUE THREAD

**Issue by ildarkhasanshin** [akalongman/sublimetext-codeformatter#238]
how to set up in one line?
in the example below in reference appears extra space because that tag <a> on three lines

https://s.mail.ru/MBKm/NN5JShizc
https://s.mail.ru/LA6u/wuXutwsbd

**Comment by RevanProdigalKnight** [comment]
I can't view the images that you provided, the page is giving me an error of some sort.

**Comment by ildarkhasanshin** [comment]
![img-2016-06-10-12-42-45](https://cloud.githubusercontent.com/assets/8765305/15971546/640bec40-2f5b-11e6-9403-cab91ee5c851.png)
![img-2016-06-10-12-44-29](https://cloud.githubusercontent.com/assets/8765305/15971547/642cedfa-2f5b-11e6-83e4-2debd1617dab.png)

**Comment by RevanProdigalKnight** [comment]
To my knowledge, this is not currently supported by either of the HTML formatters, but I am working on a fix for `<textarea>` tags in the RegExp formatter that could be extended to work on any tag with only 1 line inside

**Comment by ericmagnuson** [comment]
Yes, this should really be fixed.  It's bad to put text nodes on their own lines as browsers will add an extra space before and after the text because the browser reduces all the extra whitespace down to one space.
